test/robot/report/report.proto

```diff
@@ -29,8 +30,21 @@ message Input{
   string gapit = 2;
   // Gapis is the stash id of the graphics analysis server to use.
   string gapis = 3;
+  // GapidApk is the stash id of the gapid.apk to use for a Vulkan trace.
+  string gapid_apk = 4;
+  // Layout represents the ABI details for Vulkan report.
+  ToolingLayout layout = 5;
   // Package is the stash id of the package used to generate the report.
-  string package = 4;
+  string package = 6;
+  // API is the API of the commands to report in the trace.
+  string  API = 7;
```

Comment:
Why is this capitalized?

Reply:
Changed to `api` (so the generated code will be `Api`)